app/interface/controllers/courses/attendance.py

```
@@ -0,0 +1,34 @@
+        summary="출석 목록",
+        description="해당 강좌의 출석 목록을 불러옵니다.",
+        response_class=ORJSONResponse,
+        response_model=AttendanceListResponse,
+        status_code=status.HTTP_200_OK,
+    )
+    async def get_attendance_list(
+        self,
+        course_code: Annotated[str, Path(alias="courseCode")],
+    ) -> AttendanceListResponse:
+        attendance_list = await self.usecase.get_lecture_attendance_list(
+            course_code=course_code
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)